tests/components/withings/common.py

```diff
@@ -290,7 +289,7 @@ async def unload(self, profile: ProfileConfig) -> None:
         config_entries = get_config_entries_for_user_id(self._hass, profile.user_id)
 
         for config_entry in config_entries:
-            await async_unload_entry(self._hass, config_entry)
+            await config_entry.async_unload(self._hass)
```

Comment:
Using `async_unload_entry` directly meant that the config entry was still marked as LOADED.